helix-view/Cargo.toml

```diff
@@ -22,6 +22,7 @@ helix-lsp = { version = "0.6", path = "../helix-lsp" }
 helix-dap = { version = "0.6", path = "../helix-dap" }
 crossterm = { version = "0.25", optional = true }
 helix-vcs = { version = "0.6", path = "../helix-vcs" }
+libc = "0.2"
```

Comment:
Should this be listed under `[target.'cfg(unix)'.dependencies`? It probably doesn't make a difference

Reply:
Thanks, yes, you're right, it's a unix only dependency. I've addressed this now.